flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Utils/NXSystemMessages.js

```diff
@@ -72,5 +72,6 @@ const NXFictionalMessages = {
     noRefWpt:               new McduMessage("NO REF WAYPOINT", false, false),
     noWptInfos:             new McduMessage("NO WAYPOINT INFOS", false, false),
     emptyMessage:           new McduMessage(""),
-    weightUnitChanged:      new McduMessage("UNIT CHANGED RELOAD A/C", true, true)
+    weightUnitChanged:      new McduMessage("UNIT CHANGED RELOAD A/C", true, true),
```

Comment:
I suggest removing the `weightUnitChanged` and using the new `reloadPlaneApply` because they both have the same intention.

Reply:
This isn't used in this, out of scope for this PR, that's for weights lb/kg.

Reply:
MisterChocker is saying share the same reload message for both changing units, and changing the keyboard setting... i.e. delete the unit changed one and replace it with the new more general one.

Reply:
yes but this PR is the one that adds the reloadPlaneApply NXSystemMessage, weightUnitChanged is used in the ECAM so if I simply deleted it it would cause an exception there.

Reply:
This has been changed in the latest commit